src/Components/Components/src/ChangeEventArgs.cs

```diff
@@ -13,7 +13,6 @@ public class ChangeEventArgs : EventArgs
         /// <summary>
         /// Gets or sets the new value.
         /// </summary>
-        public object Value { get; set; }
-
+        public object Value { get; set; } = default!;
```

Comment:
This probably reflects my lack of experience with the nullability checking, but `default!` is a way of setting `Value` to null but telling the compiler to pretend it's not-null, right?

Why is this preferable compared with declaring the type as nullable? Is it because you reason that the framework will always provide a nonnull value by the time it gets into user code?

Reply:
You answered your own question :)

Reply:
I went back and it looks like there are code paths this could be null. So this would be a vanilla nullable value. As @javiercn pointed out, we generally want to express the nominal nullability especially for values that the framework sets up. It just makes using the framework a lot more reasonable.